pkg/imgpkg/bundle/images_lock.go

```diff
@@ -26,6 +25,13 @@ type ImagesLock struct {
 	relativeToRepo string
 }
 
+func (o *ImagesLock) generateImagesLocations() {
+	for i, imgRef := range o.ImageRefs() {
```

Comment:
lets do `range o.imagesLock.Images` so that it's clear it's related to what we are mutating two lines down (`o.imagesLock.Images[i]...`).